tests/aws/test_multi_accounts.py

```diff
@@ -1,25 +1,10 @@
-import pytest
-
 from localstack.testing.pytest import markers
 from localstack.utils.strings import short_uid
 
 
-@pytest.fixture
```

Comment:
This was moved to `tests/conftest.py`, so that I could reuse it for all the new multi-region tests.

Reply:
In general, you can also use the fixture `aws_client_factory` directly, like this:
```python
aws_client_factory(region_name=region_name, aws_access_key_id=access_key_id).sqs
```
This properly preserves the typing. I know the multi account tests do it differently, but generally the direct usage way would be preferred.

Reply:
Done - all tests now use `aws_client_factory`. Thanks for the hint.

I've reverted the change to the multi account tests as well.